__tests__/app.test.js

```diff
@@ -78,6 +78,8 @@ describe('GET /api/articles/:article_id', () => {
             expect(article instanceof Object).toBe(true);
             expect(article).toHaveProperty('article_id', 2);
             expect(article).toHaveProperty('title', 'Sony Vaio; or, The Laptop');
+            expect(article).toHaveProperty('comment_count', '0');
```

Comment:
This shouldn't be added to an old test, but should be in it's own additional test block, please. The existing test proved what it needed to, and should remain as it was in order to show how the project has progessed.